baseapp/params.go

```diff
@@ -18,6 +18,7 @@ var (
 	ParamStoreKeyBlockParams     = []byte("BlockParams")
 	ParamStoreKeyEvidenceParams  = []byte("EvidenceParams")
 	ParamStoreKeyValidatorParams = []byte("ValidatorParams")
+	ParamStoreKeyVersionParams   = []byte("VersionParams")
```

Comment:
I forget, are these accessible by the gov handler? are we going to have to add this to the params filter? If so we should create an issue ofc

Reply:
Yes we should. Good catch. Are the other params here protected by the paramsfilter